doc/classes/Window.xml

```diff
@@ -106,12 +111,14 @@
 			<return type="void">
 			</return>
 			<description>
+				Causes the window to attain focus, allowing it to receive user-input.
```

Comment:
I'm not sure "attain focus" is correct, sounds mystical :) ("attain nirvana")

"grab" or "take" would be clearer and easier on non-native speakers.